### PR TITLE
The first capstone project

### PR DESCRIPTION
**Pull request by omar25ahmed**
### The Capstone Project
**Day 1:**
- Finishing the structure of the page.
- style the HTML.
- adding functionality to the menu.
- add the images.

**Day 2:**
- Finishing the mobile version of the two pages.
- Finishing the Desktop Version of the Home page.

**Day 3:**
- Finishing the about page Desktop Version.
- Adding feature speakers section Dynamically by JavaScript.

Loom Video: https://www.loom.com/share/4266a49d0578406f8aaca90eaae7e727

### CODE REVIEW

File: About/style.css [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+  /* margin: 50px 10px 0 50px;
+  width: 40%;
+  padding-left: 0;
+  height: 200px; */
+}
+
+/* .past img {
+  height: 200px;
+  object-fit: cover;
+} */
```
Comment (Hamzaoutdoors) [inline About/style.css]: - [ ] Kindly remove unnecessary comments from code that will make your code readable and clear

File: README.md [hunk 2]
```diff
@@ -6,10 +6,11 @@ My First Capstone Project
 This project is my first Captone and this is the first project to work on a project for a customer that wants to make a website to announce an event.
 
 ## This is an image for it
-
+![Screenshot from 2022-02-03 00-54-00](https://user-images.githubusercontent.com/92755394/152251439-c8655685-3403-4fa2-91e1-1888fd7573e7.png)
+![Screenshot from 2022-02-03 00-54-14](https://user-images.githubusercontent.com/92755394/152251446-e1484cb9-04dd-41e5-bab7-7032933cba8e.png)
 
 ## Live Demo
-
+[Live Demo Here](https://omar25ahmed.github.io/Omar-Ragheb-First-Capstone/)
```
Comment (Hamzaoutdoors) [inline README.md:16]: - [ ] The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/), Please, Add this in your readme file

**Review by Hamzaoutdoors — CHANGES_REQUESTED** [review]
Hey @omar25ahmed, I am **Hamza** your reviewer for this PR 😸 

_You did a great job, I really enjoyed surfing on your fancy website 😃_ 

## To Highlight
✔️ The repo looks professional
✔️ Git/GitHub flow best practices have been followed
✔️ There are no linter errors
✔️ You have a descriptive README
✔️ Wonderful UI design and good choice of theme🥇 
✔️ Great Job building your project with **Mobile-First**

There are some issues that you still need to work on to go to the next project but you are almost there!

image

## Required Changes ♻️

- [REQUIRED] 

You were asked to record a `video `presenting the features of your project, and submit the link for your presentation in the README file, unfortunately, I can't see the link for your presentation, Kindly try to record a video for a maximum of 5min before submitting the project for the second review

_See the inline comments below for more required changes 😃_  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
-  
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
## Hi @omar25ahmed, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**